frontend/src/app/modules/i18n/i18n.module.ts

```diff
@@ -48,13 +48,13 @@ export class I18nModule {
       translateCacheService.init();
     }
 
-    translate.addLangs(['en', 'ru']);
+    translate.addLangs(['en', 'ru', 'fr', 'gr']);
```

Comment:
Array is some kind of magic values. Maybe move it out in constants? Because it is being repeated in some other places. One time someone may forget to add new lang somewhere

Reply:
It's not a magic value at all, because its meaning is obvious. But it is being repeated nevertheless

Reply:
ok :)